akka-remote/src/main/scala/akka/remote/transport/netty/NettySSLSupport.scala

```diff
@@ -66,10 +69,16 @@ private[akka] class SSLSettings(config: Config) {
         factory.init(loadKeystore(SSLKeyStore, SSLKeyStorePassword), SSLKeyPassword.toCharArray)
         factory.getKeyManagers
       }
-      val trustManagers = {
+      val trustManagers = if (SSLTrustManagerFactoryClass.isEmpty) {
```

Comment:
We could also implement the former trust manager factory using the new method but I guess keeping it like this is a bit more performant for the common case. :+1:

Reply:
Yea, but the loadKeystore internal function was so neat and I didn't immediately see how I would extract it  and reuse it, but I do like the idea of having one way of instantiating the factory and use it.

Reply:
I changed it so there is one instantiation mechanism. No special cases to test. One approach with a default and possibility to specialise.